### DISCUSSION TITLE
how to quit without press ENTER or command

### DISCUSSION THREAD

**Discussion by davidgao7** [LazyVim/LazyVim#3482] [Q&A]
Hello everyone, 

after I do sync,  if I want to quit at the dashboard, this message will appear;
if I open a file and quit, the iterm2 tmux window will lag 1 second, there's an error message appear swiftly during the lag, and then I'm back to command line

does anyone have a similar experience?

<img width="725" alt="Screenshot 2024-06-05 at 9 46 52 PM" src="https://github.com/LazyVim/LazyVim/assets/39502459/d3fc200d-0716-4e6a-9347-c27eaeecec52">

**Comment by folke** [comment]
That's because an error happens on quit. Update noice and you will probably see the error.
If you're on windows, then it's likely the broke shada implementation.

**Reply by davidgao7** [reply]
Thanks for replying man, I tried to record the screen and find the frame
<img width="1183" alt="Screenshot 2024-06-06 at 3 27 48 PM" src="https://github.com/LazyVim/LazyVim/assets/39502459/6c24a9a7-7dc0-4523-b85f-8fa3ed541bd8">

**Reply by davidgao7** [reply]
I think semshi has been removed...? I'm not sure, I checked lazy plugin manager and semshi is not showing

<img width="1640" alt="Screenshot 2024-06-06 at 3 34 08 PM" src="https://github.com/LazyVim/LazyVim/assets/39502459/67406b93-af48-48c1-97a2-7a2fb8de40a0">

**Reply by dpetka2001** [reply]
Yes, it's been removed as an Extra. So, users have to install it themselves if they want. Or don't use it at all.

**Reply by davidgao7** [reply]
Thank you!, I disable it in lazy.lua, the error is gone now, but when I quit I can still see this line
<img width="1358" alt="Screenshot 2024-06-06 at 3 47 49 PM" src="https://github.com/LazyVim/LazyVim/assets/39502459/cc58e6d0-7545-424f-81f2-b3a169e5567f">

**Reply by dpetka2001** [reply]
See this [comment](https://github.com/LazyVim/LazyVim/issues/3456#issuecomment-2149087836). Maybe it will help you. I've never used semshi so no idea really.

**Comment by Old-Farmer** [comment]
See this <https://github.com/LazyVim/LazyVim/discussions/3445> , may be helpful.

**Reply by davidgao7** [reply]
Thx for the reply!  So, does it mean semshi is somehow still in my system? how could I find it?
<img width="865" alt="Screenshot 2024-06-11 at 11 34 19 AM" src="https://github.com/LazyVim/LazyVim/assets/39502459/855dcf0f-2076-404e-9f35-d3b9ba60f3d8">

**Reply by dpetka2001** [reply]
Try to search in `~/.local/share/nvim/rplugin.vim` for any entries that might mention `semshi` and delete them.

**Reply by Old-Farmer** [reply]
Also See 'help remote_plugin.txt' if you have any questions with the `rplugin.vim` file.
